### PR TITLE
coordinator: fix integration test smart contract deploy helper

### PR DESCRIPTION
**Pull request by jpnovais**
This PR implements issue(s) #

### Checklist

* [ ] I wrote new tests for my new core changes.
* [ ] I have successfully ran tests, style checker and build against my new changes locally.
* [ ] I have informed the team of any breaking changes if there are any.

### CODE REVIEW

File: coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt [hunk 1]
```diff
@@ -75,7 +75,7 @@ fun executeCommand(
 }
 
 private val lineaRollupAddressPattern = Pattern.compile(
-  "^LineaRollup(?:AlphaV3)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
+  "^LineaRollupV(?:[0-9]+)? deployed: address=(0x[0-9a-fA-F]{40}) blockNumber=(\\d+)"
```
Comment (VGau) [inline coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt]: Can we have version like V5.1 ? If yes this regex won't work if I am not wrong :)

**Review by thedarkjester — APPROVED** [review]

File: coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt (hunk 1 above)
Comment (thedarkjester) [reply coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt]: the contract itself won't be able to be named V5.x just the field inside (contract names don't like the `.`s)

Comment (VGau) [reply coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt]: Ah yes you're right :)

**Review by VGau — APPROVED** [review]

**Review by thedarkjester — APPROVED** [review]

File: coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt (hunk 1 above)
Comment (jpnovais) [reply coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/MakefileContractDeploymentHelper.kt]: That's a good point. Let me check it out

**Review by VGau — APPROVED** [review]

**Review by thedarkjester — APPROVED** [review]

**Pull request merged**